build-support/bin/travis-ci.sh

```diff
@@ -53,4 +53,7 @@ $(list_pythons | sort -u)
 EOF
 fi
 
-exec ./build-support/bin/ci.sh "$@"
\ No newline at end of file
+stty rows 60
```

Comment:
A comment would be great as to why. I think its probably horrible that we have tests that parse output (and to boot, in this fragile a way). Calling attention to this magic as a hack to support the test hacks seems like a good idea for future cleanup expeditions to be more fearless.

Reply:
Very open to feedback on the tests in https://github.com/pantsbuild/pants/pull/6661 which are the relevant ones (which I just added).

In the mean time, I'll add a comment and merge to get master green.

Reply:
Aha. Well, in that case, setting TERM=dumb and / or COLUMNS=XYZ inside just those tests makes a lot more sense to me.